src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java

```diff
@@ -687,7 +687,8 @@ public boolean onRunningTick(ItemStack stack) {
 
         // Draw energy from GT hatches
         for (GT_MetaTileEntity_Hatch_Energy eHatch : super.mEnergyHatches) {
-            if (eHatch == null || eHatch.getBaseMetaTileEntity().isInvalidTileEntity()) {
+            if (eHatch == null || eHatch.getBaseMetaTileEntity() == null
+                    || eHatch.getBaseMetaTileEntity().isInvalidTileEntity()) {
```

Comment:
update GT and
```suggestion
            if (eHatch == null || !eHatch.isValid()) {
```

Reply:
maybe do your own PR, I dont know what that is or how that replaces the other part.
I tested my version though and it works.